files/build_templates/docker_image_ctl.j2

```diff
@@ -27,6 +27,21 @@ link_namespace() {
 }
 {%- endif %}
 
+function updateSyslogConf()
+{
+    # On multiNPU platforms, change the syslog target ip to docker0 ip to allow logs from containers
+    # running on the namespace to reach the rsyslog service running on the host
+    # Also update the container name  
+    if [[ ($NUM_ASIC -gt 1) ]]; then
+        TARGET_IP=$(ip -o -4 addr list docker0 | awk '{print $4}' | cut -d/ -f1)
+        CONTAINER_NAME="{{docker_container_name}}$DEV"
+        TMP_FILE="/tmp/rsyslog.$CONTAINER_NAME.conf"
+        sonic-cfggen  -t /usr/share/sonic/templates/rsyslog-container.conf.j2 -a "{\"target_ip\": \"$TARGET_IP\", \"container_name\": \"$CONTAINER_NAME\" }"  > $TMP_FILE
```

Comment:
>container_name [](start = 116, length = 14)

This field make original code useless.
```
ARG docker_container_name
```
Could you remove original related code in Dockerfile(s)

Reply:
Agreed.  I will do it in a separate PR. Filed [this](https://github.com/Azure/sonic-buildimage/issues/4778) issue to track this.